coq_bug_minimizer.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# usage: coq_bug_minimizer.sh 'script' comment_thread_id comment_author github_token bot_name bot_domain
+
+set -ex
+
+if [ $# != 6 ]; then >&6 echo Bad argument count; fi
+
+script=$1
+comment_thread_id=$2
+comment_author=$3
+token=$4
+bot_name=$5
+bot_domain=$6
+branch_id=$(($(od -A n -t uI -N 5 /dev/urandom | tr -d ' ')))
+repo_name="coqbot-dev/run-coq-bug-minimizer"
+branch_name="run-coq-bug-minimizer-$branch_id"
+
+wtree=$(mktemp -d)
+
+git fetch "https://github.com/$repo_name.git" "refs/heads/master:$branch_name"
+git worktree add "$wtree" "$branch_name"
+pushd "$wtree"
+
+printf "%s %s %s %s" "$comment_thread_id" "$comment_author" "$repo_name" "$branch_name" > coqbot-request-stamp
+printf "%s\n" "$script" > coqbot.sh
+sed -i 's/\r$//g' coqbot.sh
+sed -i "s/COQBOT_URL='.*'/COQBOT_URL='https:\/\/$bot_domain\/coq-bug-minimizer'/" coqbot-config.sh
```

Comment:
If we don't plan to change the other parameters, what about removing them from `coqbot-config.sh` and writing the file in its entirety instead of relying on `sed`?

Reply:
I've replaced `COQBOT_URL=''` with `COQBOT_URL="$(cat "$DIR/coqbot.url")"` in this file permanantly.  So now you either should do
```suggestion
sed -i "s/COQBOT_URL=.*/COQBOT_URL='https:\/\/$bot_domain\/coq-bug-minimizer'/" coqbot-config.sh
```
or
```suggestion
echo "https://$bot_domain/coq-bug-minimizer" > coqbot.url
```

Reply:
Picked the second one.